lib/vault/defaults.rb

```diff
@@ -121,12 +122,16 @@ def ssl_ciphers
         ENV["VAULT_SSL_CIPHERS"] || SSL_CIPHERS
       end
 
-      # The raw contents (as a string) for the pem file. To specify the path to
-      # the pem file, use {#ssl_pem_file} instead. This value is preferred over
-      # the value for {#ssl_pem_file}, if set.
+      # The raw contents (as a string) for the pem file, Base64-encoded or unencoded.
```

Comment:
This implies that the output is Base64-encoded? It's always the raw PEM file, i'd leave the comment as it was before.